Feature: Invert Block

Shriank opened the pull request:
This contains the patch for a new invert block where notes can be inverted around a pivot note. 
The StepNote Fix has been added.

walterbender commented:
I'm not sure what is going on with your fork, but this series of patches removes things from master and then adds them back. Cannot you update your fork to be in sync with my master branch, commit your invert-block related changes, and make a new pull request?

Shriank commented:
Hey Walter,

It isn't working like that since the fork changes adds a few extra lines,
all of which I have added. There are multiple changes that even I can't
keep track off all the changes that I made for the invert condition.
Is it possible to just merge the master and musicblocks3 instead?
Nonetheless, I have added all the changes from the stepNote patch. Can I
resubmit this patch?

Thanks
/Shrix
ᐧ

walterbender commented:
Can you make a fresh fork of the project? Then you can add the invertblock
changes.

On Tue, Jan 12, 2016 at 11:24 AM, Shriank <notifications@github.com> wrote:

> Hey Walter,
>
> It isn't working like that since the fork changes adds a few extra lines,
> all of which I have added. There are multiple changes that even I can't
> keep track off all the changes that I made for the invert condition.
> Is it possible to just merge the master and musicblocks3 instead?
> Nonetheless, I have added all the changes from the stepNote patch. Can I
> resubmit this patch?
>
> Thanks
> /Shrix
> ᐧ
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/walterbender/musicblocks/pull/107#issuecomment-170964243>
> .
>



-- 
Walter Bender
Sugar Labs
http://www.sugarlabs.org

Shriank commented:
OK. I shall try once more.
